packages/zoe/src/contracts/simpleExchange.js

```diff
@@ -124,6 +120,10 @@ export const makeContract = harden(zcf => {
 
   return harden({
     invite: makeExchangeInvite(),
-    publicAPI: { makeInvite: makeExchangeInvite, getBookOrders, getOffer },
+    publicAPI: {
+      makeInvite: makeExchangeInvite,
+      getUpdateSince: notifier.getUpdateSince,
```

Comment:
Do we want to use the name `getUpdateSince`? I thought `getBookOrders` was more informative.

Reply:
Ok, sounds good to me